src/getHumanAge.test.js

```diff
@@ -7,4 +7,53 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('if felines and canine the first 14 years then return'
+    + 'the array with two 0', () => {
```

Comment:
Latin terms, smart🧠 but it can require some time to google what is it for someone like me 😅